src/psyclone/dynamo0p3.py

```diff
@@ -136,10 +140,10 @@
 VALID_LOOP_TYPES = ["dofs", "colours", "colour", ""]
 
 # ---------- psyGen mappings ------------------------------------------------ #
-# Mappings used for scalar reductions by non-API-Specific code in psyGen.
+# Mappings used by non-API-Specific code in psyGen.
```

Comment:
This will ultimately be 'proper' PSyIR but that isn't going to happen for a while yet.

Reply:
I suspect I will have to find a way around it in #774, as I will have just one scalar name.